robolectric/src/test/java/org/robolectric/shadows/ShadowCanvasTest.java

```diff
@@ -529,4 +530,36 @@ public void restoreToCount() {
 
     assertThat(canvas.getSaveCount()).isEqualTo(save1);
   }
+
+  @Test
+  public void drawRect_toCanvas() {
+    Bitmap bitmap = Bitmap.createBitmap(10, 10, Bitmap.Config.ARGB_8888);
+    bitmap.eraseColor(Color.BLACK);
+    assertThat(bitmap.getPixel(0, 0)).isEqualTo(Color.BLACK);
+
+    Canvas canvas = new Canvas(bitmap);
+    Paint paint = new Paint();
+    paint.setStyle(Paint.Style.FILL);
+    paint.setAntiAlias(true);
+    paint.setColor(Color.WHITE);
+    canvas.drawRect(0, 0, 10, 10, paint);
+    assertThat(bitmap.getPixel(0, 0)).isEqualTo(Color.WHITE);
+  }
+
+  @Test
+  public void draw_Rect() throws Exception {
```

Comment:
What about using name `drawRect_withRectF`?

Reply:
i do it..